src/Persistence/MassTransit.EntityFrameworkCoreIntegration/Saga/Configuration/EntityFrameworkSagaRepository.cs

```diff
@@ -0,0 +1,62 @@
+namespace MassTransit.EntityFrameworkCoreIntegration.Saga.Configuration
+{
+    using System;
+    using System.Collections.Generic;
+    using Mappings;
+    using MassTransit.Saga;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+
+    class EntityFrameworkSagaRepository : IEntityFrameworkSagaRepository
+    {
+        readonly DbContextOptions _dbContextOptions;
+        readonly IList<ISagaClassMap> _configurations;
+
+        public EntityFrameworkSagaRepository(DbContextOptions dbContextOptions)
+        {
+            _dbContextOptions = dbContextOptions;
+            _configurations = new List<ISagaClassMap>();
+        }
+
+        public void ConfigureSaga<TSaga>(Action<EntityTypeBuilder<TSaga>> configure = null)
+            where TSaga : class, ISaga
+        {
+            _configurations.Add(new ActionSagaClassMap<TSaga>(configure));
```

Comment:
`_configurations.TryAdd(typeof(TSaga), new ActionSagaClassMap<TSaga>(configure));`
to avoid duplicate configuration

Reply:
you could merge with my repo :)